guides/release/components/helper-functions.md

```diff
@@ -315,7 +315,69 @@ components.
 ```
 
 In the component's template, you can then use the `person` object:
-
 ```handlebars {data-filename=app/components/greeting/template.hbs}
 Hello, {{@person.givenName}} {{@person.familyName}}
 ```
+
+### The `in-element` helper
+
+Using the [`{{in-element}}`](https://api.emberjs.com/ember/3.27/classes/Ember.Templates.helpers/methods/in-element?anchor=in-element) helper, you can render content into a specific DOM element that is in a different part of the page. For instance, we might want
+to render a modal, tooltip or dropdown.
```

Comment:
I used underscores to highlight the word "different." This way, I think we can remove the word "specific," which is a bit redundant.

```suggestion
Using the [`{{in-element}}`](https://api.emberjs.com/ember/3.27/classes/Ember.Templates.helpers/methods/in-element?anchor=in-element) helper, you can render content into a DOM element that is in a _different_ part of the page. For instance, we might want
to render a modal, tooltip, or dropdown.
```